src/QA/QA.Web/ViewModels/DashboardViewModel.cs

```diff
@@ -9,5 +9,12 @@ public class DashboardViewModel
         public int TotalVacanciesResubmitted { get; set; }
 
         public List<ReviewDashboardItem> AllReviews { get; set; }
+
+        public string SearchTerm { get; set; }
+
+        public string LastSearchTerm { get; set; }
+
+        public List<VacancyReviewSearchModel> SearchResults { get; set; } = new List<VacancyReviewSearchModel>();
+        public bool IsPostBack { get; set; }
```

Comment:
I don't think the view should care about postbacks.  We tend to create bool properties and helper functions in the VM to change the view.  Maybe just create a `DisplayNoResultsMessage` property in the VM instead (I think this is only dependency on knowledge of a postback)

Reply:
I think this needs wider consultation. The page is designed in unconventional way. We perhaps need to separate results page, that way we can use query string while doing a GET request. It doesn't make sense to fit it all on dashboard.